service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/AbstractControllerTest.java

```diff
@@ -195,12 +202,20 @@ AboutToStartOrSubmitCallbackResponse postSubmittedEvent(String filename) {
         return postSubmittedEvent(filename, SC_OK);
     }
 
+    LocalDateTime timeNow() {
```

Comment:
I would maybe call this one just `now` instead. Although we don't have a use for it right now we may in the future have a field that is LocalTime and then it might get confusing

Reply:
I can change it to dateTimeNow()

Reply:
actually I will change it to now() as suggested, to match the Time interface.